ocr.py

```diff
@@ -44,3 +49,116 @@ def ocr(self, image_file, show_steps=False):
         features = self.feature_extractor.extract(image_file.image, segments)
         classes = self.classifier.classify(features)
         return classes, segments
+
+
+class SimpleOCR(object):
+    def __init__(self, segmentation="contour", extraction="simple", classification="knn",
```

Comment:
Definitely feel free to mess with the code if you feel it makes the overall thing clearer - this is pretty old, and was never meant to be used for anything serious, so it can use some improvements.

All the functionality you added seems quite useful, so I think we can merge all of it. It'll make it easier to reuse the existing code as well.